### PR TITLE
AT_18.01_001 | Profile Page | Verify user profile name redirect.

### PR DESCRIPTION
**Pull request by TatiChap**

### CODE REVIEW

**Review by VolhaChudzesa — APPROVED** [review]

File: cypress/e2e/userProfilePage.cy.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/// <reference types="cypress"/>
+
+describe("User Profile Page", () => {
+  it("AT_18.01_001 | <Profile Page> Verify user profile name redirect.", function () {
```
Comment (valentin-kushnarev) [inline cypress/e2e/userProfilePage.cy.js:4]: Hello! Remove the arrows <> from title.

Comment (TatiChap) [reply cypress/e2e/userProfilePage.cy.js]: Hello! Fixed. Please, reconsider.

**Review by milladiz84 — COMMENTED** [review]
Hi, could you add the AT Trello link here

**Comment by TatiChap** [comment]
https://trello.com/c/5aK5lyiw

**Comment by TatiChap** [comment]
Hi, @milladiz84 done! Please, reconsider.

**Review by milladiz84 — APPROVED** [review]

File: cypress/e2e/userProfilePage.cy.js (hunk 1 above)
Comment (valentin-kushnarev) [reply cypress/e2e/userProfilePage.cy.js]: Ok, I will approve after CI.

**Review by milladiz84 — COMMENTED** [review]
Could you attach your TC to the AC in Trello. 

Cards should be connected:
US-TC
TC-AT
AT-US

**Comment by TatiChap** [comment]
@milladiz84 Great point, thanks! Fixed.

**Review by SashaFoxx — APPROVED** [review]

File: cypress/e2e/userProfilePage.cy.js (hunk 1 above)
Comment (TatiChap) [reply cypress/e2e/userProfilePage.cy.js]: Thanks)

**Review by valentin-kushnarev — APPROVED** [review]

**Pull request merged**